src/data/finsupp/basic.lean

```diff
@@ -337,6 +337,11 @@ begin
   exact nontrivial_of_ne (single default x) 0 (mt single_eq_zero.1 hx)
 end
 
+-- really we only need one infinite and the others inhabited
```

Comment:
I don't get what this remark is referring to. If you mean `α` and `M`, note that `ℕ →₀ fin 1` is not infinite, even though `ℕ` is infinite and `fin 1` is inhabited.

Reply:
This is a copy paste from the other one, but it doesn't really make sense here, it would for dfinsupp as I mean we only need one `i` such that `\pi i` is infinite, the others only need to be inhabited